Assets/Scripts/UserConfig.cs

```diff
@@ -425,6 +426,17 @@ public float OdsPoleCollapsing
                     }
                 }
             }
+
```

Comment:
I wasn't sure if `Video` was the right section for this, but I didn't see anything better. It seems like it's mostly video out stuff...